src/Illuminate/Foundation/Testing/Concerns/MocksApplicationServices.php

```diff
@@ -88,7 +89,14 @@ protected function withoutEvents()
             $this->firedEvents[] = $called;
         });
 
+        $mock->shouldReceive('until')->andReturnUsing(function ($called) {
+            $this->firedEvents[] = $called;
+
+            return true;
+        });
+
         $this->app->instance('events', $mock);
+        Model::setEventDispatcher($mock);
```

Comment:
Won't this mock bleed over into subsequent tests that are run since Eloquent holds onto a static copy of it?